src/mixins/resource-lookup-mixin.ts

```diff
@@ -46,37 +44,45 @@ export default class ResourceLookupMixin extends Vue {
     directorWarning (directors: Array<any>): AlertMessageIF {
       const filingCode = this.isBComp ? FilingCodes.DIRECTOR_CHANGE_BC : FilingCodes.DIRECTOR_CHANGE_OT
       // FUTURE: Too much code for this. Can be condensed and made more reusable.
-      if (directors && directors.length) {
-        const configSection = this.configObject.flows.find(x => x.feeCode === filingCode).warnings
+      if (directors?.length) {
+        const warnings = this.configObject?.flows?.find(x => x.feeCode === filingCode)?.warnings
         let errors = []
         // If this entity has a BC Residency requirement for directors, one of the
         // directors specified needs to have both their mailing and delivery address within British Columbia
-        if (configSection.bcResident) {
+        if (warnings?.bcResident) {
```

Comment:
This line (and the 2 below) was the real culprit: the filing code did not have any warnings, so de-referencing the property caused an exception, causing the COD page to fail to load.
This code no longer fails, even if the db has invalid legal type.